res/values/strings.xml

```diff
@@ -396,11 +400,20 @@
     <string name="mailing_list">Mailing List</string>
     <string name="mailing_list_profile_info">Changes to mailing list name and image apply on this device only.</string>
 
+    <!-- broadcast lists -->
+    <string name="create_broadcast_list">Create Broadcast List</string>
+    <string name="menu_edit_broadcast_list">Edit Broadcast List</string>
+    <string name="menu_broadcast_list_name">Broadcast List Name</string>
+    <string name="broadcast_please_enter_broadcast_list_name">Please enter a name for the broadcast list.</string>
+    <string name="broadcast_list_warning">Broadcast Lists are always unencrypted because they use blind copy (BCC)</string>
+
     <!-- webxdc -->
     <!-- "Start..." button for an app -->
     <string name="start_app">Start…</string>
     <!-- this is a warning that is shown when one tries to send something to a chat that is not yet accepted. -->
     <string name="accept_request_first">Please accept the chat request first.</string>
+    <string name="show_webxdc_in_chat">Show Private App in Chat</string>
```

Comment:
already used in the same menu:
<img width="185" alt="Bildschirmfoto 2022-11-16 um 18 37 59" src="https://user-images.githubusercontent.com/18725968/202253188-0200715a-e958-4447-8e57-5b7b06ebbc27.png">

right click on webxdc message in chat audit dialog

Reply:
i understand. wondering if that option is really needed, one can easily go to the private app from the info-message (we do not have shortcuts for images etc. as well). so, tbh, i would just remove it.

however, sure, no blocker.

nb: in general, menu items should _never_ contain line breaks.

Reply:
> nb: in general, menu items should never contain line breaks.

so we need a desktop issue to make the context menu larger in case the text is too long.